src/components/carousel/carousel.module.css

```diff
@@ -0,0 +1,73 @@
+/* css for carousel below */
```

Comment:
Just wanted to know. what is the use of `module.css` extension?

Reply:
The css modules are used to provide scope to the css files, this way redundant classnames in different css files doesnt overrride each other.